utils/sl/check_script.py

```diff
@@ -135,17 +136,20 @@ def write_variables(outfile, prefix, variables):
                 outfile.write("%s%s = None\n" % (prefix, variable))
 
         outfile.write("# pylint: enable=invalid-name\n")
+        outfile.write("# pylint: enable=import-outside-toplevel\n")
         return newlines
 
     @staticmethod
     def write_header(outfile):
         """write the header"""
         outfile.write("""# pylint: disable=wildcard-import
+# pylint: disable=import-outside-toplevel
```

Comment:
This is a top-level import isn't it so why do we need to disable it?

Reply:
The header should get written first though it does look like if the file for some reason doesn't start with # or """ blocks, it could defer til first occurrence of Import or Export

Reply:
You don't want to change the scope though.